components/script/dom/webglrenderbuffer.rs

```diff
@@ -24,6 +26,8 @@ pub struct WebGLRenderbuffer {
     size: Cell<Option<(i32, i32)>>,
     internal_format: Cell<Option<u32>>,
     is_initialized: Cell<bool>,
+    // Framebuffer that this texture is attached to.
+    attached_framebuffers: DomRefCell<Vec<Dom<WebGLFramebuffer>>>,
```

Comment:
I don't understand why this is needed and why we can't just keep track of a refcount.